src/OpenTelemetry/docker-compose.yaml

```diff
@@ -0,0 +1,8 @@
+services:
+  aspire-dashboard:
+    environment:
+      Dashboard__Frontend__AuthMode: Unsecured
+    image: mcr.microsoft.com/dotnet/aspire-dashboard:8.0
```

Comment:
I did see that the python samples are using the Jaeger image and not the Aspire Dashboard, which is developed by Microsoft: https://github.com/temporalio/samples-python/tree/main/open_telemetry

I can change to the Jaeger image if desired.

The main benefit of using the Aspire dashboard over Jaeger is that it provides visualizations for metrics, which iirc Jaeger does not. I think using a Microsoft image is _probably_ okay specifically for .NET

Reply:
I don't mind this, I like the metrics. Alternatively, you can get rid of docker compose and just tell people how to start the this docker container directly in the README, but this is fine too.

Reply:
Imo, docker compose is easier with a single command, since environment variables and port forwarding are evolved